sdk/ml/azure-ai-ml/azure/ai/ml/operations/_featureset_operations.py

```diff
@@ -0,0 +1,227 @@
+from azure.ai.ml.entities._assets import Featureset
+from azure.ai.ml._utils._experimental import experimental
+from azure.core.polling import LROPoller
+from azure.core.paging import ItemPaged
+
+ops_logger = OpsLogger(__name__)
+module_logger = ops_logger.module_logger
+
+
+@experimental
+class FeaturesetOperations(_ScopeDependentOperations):
+    def __init__(
+        self,
+        operation_scope: OperationScope,
+        operation_config: OperationConfig,
+        service_client: ServiceClient022023Preview,
+        **kwargs: Dict,
+    ):
+
+        super(FeaturesetOperations, self).__init__(operation_scope, operation_config)
+        # ops_logger.update_info(kwargs)
+        self._operation = service_client.featureset_versions
+        self._container_operation = service_client.featureset_containers
+        self._service_client = service_client
+        self._init_kwargs = kwargs
+
+        # Maps a label to a function which given an asset name,
+        # returns the asset associated with the label
+        self._managed_label_resolver = {"latest": self._get_latest_version}
+
+    # @monitor_with_activity(logger, "Featureset.List", ActivityType.PUBLICAPI)
+    def list(
+        self,
+        *,
+        name: Optional[str] = None,
+        list_view_type: ListViewType = ListViewType.ACTIVE_ONLY,
+    ) -> ItemPaged[Featureset]:
+        """List the featureset assets of the workspace.
+
+        :param name: Name of a specific featureset asset, optional.
+        :type name: Optional[str]
+        :param list_view_type: View type for including/excluding (for example) archived featureset assets.
+        Default: ACTIVE_ONLY.
+        :type list_view_type: Optional[ListViewType]
+        :return: An iterator like instance of Featureset objects
+        :rtype: ~azure.core.paging.ItemPaged[Featureset]
+        """
+        if name:
+            return self._operation.list(
+                workspace_name=self._workspace_name,
+                name=name,
+                cls=lambda objs: [Featureset._from_rest_object(obj) for obj in objs],
+                list_view_type=list_view_type,
+                **self._scope_kwargs,
+            )
+        return self._container_operation.list(
+            workspace_name=self._workspace_name,
+            cls=lambda objs: [Featureset._from_container_rest_object(obj) for obj in objs],
+            list_view_type=list_view_type,
+            **self._scope_kwargs,
+        )
+
+    def _get(self, name: str, version: str = None) -> FeaturesetVersion:
+        return self._operation.get(
+            resource_group_name=self._resource_group_name,
+            workspace_name=self._workspace_name,
+            name=name,
+            version=version,
+            **self._init_kwargs,
+        )
+
+    # @monitor_with_activity(logger, "Featureset.Get", ActivityType.PUBLICAPI)
+    def get(self, *, name: str, version: Optional[str] = None, label: Optional[str] = None) -> Featureset:
```

Comment:
We do not define "label" for featureset and FeaturestoreEntity.

Reply:
okay I can remove it.
It is general pattern for other asset. @qjxu should I remove it?

Reply:
We can remove it in next PR once get inputs from other people.